File: MyHeart.DTO/Client/Client_QuestionaireResultDTO.cs
using MyHeart.DTO.Client;
using MyHeart.DTO.ProfessionalInformation;
using MyHeart.DTO.ProfessionInformation;
using System;
using System.Collections.Generic;
using System.Text;

namespace MyHeart.DTO
{
    public class Client_QuestionaireResultDTO
    {
        public ICollection<Client_QuestionaireDiseaseAverageDTO> ProbableDiseases { get; set; }
        public ICollection<Client_QuestionaireDiseaseAverageDTO> PossibleDiseases { get; set; }
        public ICollection<Client_QuestionaireDiseaseAverageDTO> ImprobableDiseases { get; set; }
    }
}

File: MyHeart.Web/ClientApp/models/questionaire/ClientQuestionaireResultDTO.ts
import ClientQuestionaireDiseaseAverageDTO from "./ClientQuestionaireDiseaseAverageDTO";


export default class ClientQuestionaireResultDTO {
    probableDiseases: ClientQuestionaireDiseaseAverageDTO[];
    possibleDiseases: ClientQuestionaireDiseaseAverageDTO[];
    improbableDiseases: ClientQuestionaireDiseaseAverageDTO[];
}